File: src/main/java/models/Livreur.java
package models;

public class Livreur {
    private int id;
    private String nom, prenom , region;

    private String email;

    private int telephone;

    public Livreur(int id, String nom, String prenom, String region, String email, int telephone) {
        this.id = id;
        this.nom = nom;
        this.prenom = prenom;
        this.region = region;
        this.email=email;
        this.telephone=telephone;
    }

    public Livreur( String nom, String prenom , String region ) {
        this.nom = nom;
        this.prenom = prenom;
        this.region = region;
        this.email=email;
        this.telephone=telephone;
    }

    public Livreur() {}

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getNom() {
        return nom;
    }

    public void setNom(String nom) {
        this.nom = nom;
    }

    public String getPrenom() {
        return prenom;
    }

    public void setPrenom(String prenom) {
        this.prenom = prenom;
    }

    public String getRegion() {
        return region;
    }

    public void setRegion(String region) {
        this.region = region;
    }

    public String getEmail() {
        return email;
    }

    public void setEmail(String email) {
        this.email = email;
    }

    public int getTelephone() {
        return telephone;
    }

    public void setTelephone(int telephone) {
        this.telephone = telephone;
    }

    @Override
    public String toString() {
        return "Livreur{" +
                "id=" + id +
                ", nom='" + nom + '\'' +
                ", prenom='" + prenom + '\'' +
                ", region='" + region + '\'' +
                ", email='" + email + '\'' +
                ", telephone=" + telephone +
                '}';
    }
}

File: src/Entity/Livreur.php
<?php

namespace App\Entity;
use App\Repository\GaleriesRepository;
use Symfony\Component\Validator\Constraints as Assert;
use Doctrine\ORM\Mapping as ORM;
use App\Repository\LivreurRepository;

#[ORM\Entity(repositoryClass: LivreurRepository::class)]

class Livreur
{

    #[ORM\Id]
    #[ORM\GeneratedValue]
    #[ORM\Column]
    private $id;

    #[ORM\Column(length:255)]
    #[Assert\NotBlank(message: "Name is required !")]
    private $nom;
    #[ORM\Column(length:255)]
    #[Assert\NotBlank(message: "Name is required !")]
    private $prenom;
    #[ORM\Column(length:255)]
    #[Assert\NotBlank(message: "Name is required !")]
    private $region;
    #[ORM\Column(length:255)]
    #[Assert\NotBlank(message: "Name is required !")]
    private $email;

    #[ORM\Column]
    #[Assert\Positive]
    
    private $telephone;

    public function getId(): ?int
    {
        return $this->id;
    }

    public function getNom(): ?string
    {
        return $this->nom;
    }

    public function setNom(string $nom): self
    {
        $this->nom = $nom;

        return $this;
    }

    public function getPrenom(): ?string
    {
        return $this->prenom;
    }

    public function setPrenom(string $prenom): self
    {
        $this->prenom = $prenom;

        return $this;
    }

    public function getRegion(): ?string
    {
        return $this->region;
    }

    public function setRegion(string $region): self
    {
        $this->region = $region;

        return $this;
    }

    public function getEmail(): ?string
    {
        return $this->email;
    }

    public function setEmail(string $email): self
    {
        $this->email = $email;

        return $this;
    }

    public function getTelephone(): ?int
    {
        return $this->telephone;
    }

    public function setTelephone(int $telephone): self
    {
        $this->telephone = $telephone;

        return $this;
    }
}
